API/index.js

```diff
@@ -51,33 +52,57 @@ class API {
     this.cloudRecording = new CloudRecording({ api: this });
     this.meeting = new Meeting({ api: this });
     this.user = new User({ api: this });
+
+    // Add throttle rules from ZOOM_THROTTLE_LIMIT_RULES file
+    THROTTLE_LIMIT_RULES.forEach((rule) => { this._addRule(rule); });
```

Comment:
Can this be an option in the constructor?

if `dontUseDefaultThrottleRules` is true, don't do line 57.